statistics.md

```diff
@@ -6,10 +6,26 @@ aliases: ['/docs/dev/statistics/','/docs/dev/reference/performance/statistics/']
 
 # Introduction to Statistics
 
-Based on the statistics, the TiDB optimizer chooses the most efficient query execution plan. The statistics collect table-level and column-level information.
+In TiDB, the statistical information you need to maintain includes the total number of rows in the table, the equal-depth histogram of columns, Count-Min Sketch, the number of `Null`s, the average length, the number of different values, etc. This document briefly introduces the histogram and Count-Min Sketch, and details the collection and maintenance of statistical information.
 
-- The statistics of a table include the total number of rows and the number of updated rows.
-- The statistics of a column include the number of different values, the number of `NULL`, the histogram, the `TOPN` value that occurs most frequently in the column, and the Count-Min Sketch of the column.
+## Histogram
+
+A histogram is an approximate representation of the distribution of data. It divides the entire range of values into a series of buckets, and uses simple data to describe each bucket, such as the number of values ​​falling in the bucket. In TiDB, an equal-depth histogram is created for the specific columns of each table. The equal-depth histogram can be used to estimate the interval query.
+
+Here "equal-depth" means that the number of values ​​falling into each bucket is as equal as possible. For example, for a given set {1.6, 1.9, 1.9, 2.0, 2.4, 2.6, 2.7, 2.7, 2.8, 2.9, 3.4, 3.5}, you want to generate 4 buckets. The equal-depth histogram is as follows. It contains four buckets [1.6, 1.9], [2.0, 2.6], [2.7, 2.8], [2.9, 3.5]. The bucket depth is 3.
+
+![Equal-depth Histogram Example](/media/statistics-1.png)
+
+For details about the parameter that determines the upper limit to the number of histogram buckets, refer to [Manual Collection](#manual-collection). When the number of buckets is larger, the accuracy of the histogram is higher; however, higher accuracy is at the cost of the usage of memory resources. You can adjust this number appropriately according to the actual scenario.
+
+## Count-Min Sketch
+
+Count-Min Sketch is a hash structure. When an equivalent query contains `a = 1` or `IN` query (for example, `a in (1, 2, 3)`), TiDB uses this data structure for estimation.
```

Comment:
```suggestion
Count-Min Sketch is a hash structure. When an equivalence query contains `a = 1` or `IN` query (for example, `a in (1, 2, 3)`), TiDB uses this data structure for estimation.
```

IMO, `equivalent query` refers to sql statements that takes the same effect, but `equivalence query` refers to a query that contains a `=` (and the like) as its condition. What do you think?

Reply:
Confirmed in https://logicalread.com/oracle-11g-hash-indexes-mc02/#.XxVFRUczZPY and https://docs.microsoft.com/en-us/dotnet/csharp/linq/, `equivalence query` is the right expression